cmd/suite_test.go

```diff
@@ -0,0 +1,155 @@
+package cmd
```

Comment:
Now that both `ui_test` and `root_test` use this code, I moved it into a `suite_test.go`. Is that odd, or does it make sense?

Reply:
Sounds good! I confess I'm not sure sticking these helpers in `suite_test` is the most idiomatic pattern (perhaps `helpers_test` would be preferable?), though I suppose this is all relatively easy to change down the road. So, if you're comfortable with `suite_test`, I'll leave that as-is.